proxy_test.go

```diff
@@ -334,3 +367,42 @@ func (suite *ProxyTests) TestMessageFormatting() {
 	assert.IsType(suite.T(), Event{}, msg)
 	assert.Equal(suite.T(), "upload", msg.Operation)
 }
+
+func (suite *ProxyTests) TestDatabaseConnection() {
+
+	// Start proxy that allows everything
+	proxy := NewProxy(suite.S3conf, NewAlwaysAllow(), suite.messenger, suite.database, new(tls.Config))
```

Comment:
```suggestion
	database, err := common.NewSDAdb(suite.DBConf)
	if err != nil {
		suite.T().Skip("skip TestShutdown since broker not present")
	}

	// Start proxy that allows everything
	proxy := NewProxy(suite.S3conf, NewAlwaysAllow(), suite.messenger, database, new(tls.Config))
```